RELEASE_NOTES.md

```diff
@@ -14,9 +14,11 @@
 
 * The `ExponentialBackoff` and `LinearBackoff` classes now require keyword arguments for their constructor. This change was made to make the classes easier to use and to avoid confusion with the order of the arguments.
 
+- HTTP2 keep-alive is now enabled by default, with an interval of 60 seconds between pings, and a 20 second timeout for responses from the service.  These values are configurable and may be updated based on specific requirements.
```

Comment:
```suggestion
- HTTP2 keep-alive is now enabled by default, with an interval of 60 seconds between pings, and a 20 seconds timeout for responses from the service.  These values are configurable and may be updated based on specific requirements.
```

Reply:
I think when used as an adjective, it should be singular.  "20 seconds" is a noun, can't have two nouns after each other.

Reply:
alright, I didn't know that :angel: